app/lib/katello/resources/candlepin.rb

```diff
@@ -386,15 +386,13 @@ def imports(organization_name)
             ::Katello::Util::Data.array_with_indifferent_access JSON.parse(imports_json)
           end
 
-          def pools(key, filter = {})
-            if key
+          def pools(owner_label, filter = {})
+            if owner_label
               # hash_to_query escapes the ":!" to "%3A%21" which candlepin rejects
               params = hash_to_query(filter)
               params += params == '?' ? '' : '&'
               params += 'attribute=unmapped_guests_only:!true'
-              json_str = self.get(join_path(path(key), 'pools') + params, self.default_headers).body
-            else
-              json_str = self.get(join_path('candlepin', 'pools') + hash_to_query(filter), self.default_headers).body
+              json_str = self.get(join_path(path(owner_label), 'pools') + params, self.default_headers).body
```

Comment:
This is within Owner, so path() here is /owners/ID/    I could change this to Pool.path(nil, owner_label) though